docker-compose.yml

```diff
@@ -100,7 +100,12 @@ services:
           3WJ4bVSmp3MOQ2EF4zrAl4fXEd4lS3JxKtzbHFbSogAmPFnuP8wIDWTDgfEq
           -----END RSA PRIVATE KEY-----
       - NHAIS_MESH_CYPHER_TO_MAILBOX=XX11=nhais_mailbox
-          TES5=nhais_mailbox
+        TES5=nhais_mailbox
+        YY21=nhais_mailbox
+        KC01=nhais_mailbox
+        WHI5=nhais_mailbox
+        SO01=nhais_mailbox
+        UNY5=nhais_mailbox
```

Comment:
Isn't there lack of braces? Or the braces are not necessary? Doest it work properly?

Reply:
It's `docker-compose.yml` file. You've mistaken it with `application.yml`